packages/components/psammead-caption/package.json

```
@@ -21,7 +21,7 @@
     "@bbc/psammead-styles": "^0.3.2"
   },
   "devDependencies": {
-    "@bbc/psammead-inline-link": "^0.3.4",
+    "@bbc/psammead-inline-link": "^1.0.0",
```

```suggestion
    "@bbc/psammead-inline-link": "^1.1.0",
```

Bump inline-link after https://github.com/bbc/psammead/pull/707 is merged and published